mdn/src/main/java/org/apache/james/mdn/MDN.java

```diff
@@ -93,8 +94,9 @@ public MDNReport getReport() {
     }
 
     public MimeMultipart asMultipart() throws MessagingException {
-        MimeMultipart multipart = new MimeMultipart();
+        MimeMultipartReport multipart = new MimeMultipartReport();
         multipart.setSubType(REPORT_SUB_TYPE);
+        multipart.setReportType("disposition-notification");
```

Comment:
We may introduce an enum for report type.

Reply:
Hmmm I more prefer a static field for this. (An enum makes sens IMO in the Mime handling library? Here it is javax.mail so we cannot change it. and do not allow much extension. Do we know all the possible report types?